bot/commands/utility/remind.js

```diff
@@ -0,0 +1,68 @@
+const source = require('rfr');
+const moment = require('moment');
+
+const logger = source('bot/utils/logger');
+const { sendMessage } = source('bot/utils/util');
+const agenda = source('scheduler');
+
+// takes a crontab formatted string 6+ arguments, first
+function reminderHandler(message, args) {
```

Comment:
Imo call this function remind or remindHandler, or change the name of the file to reminder/reminderHandler

Reply:
fixed